### ISSUE TITLE
selector-attribute-operator-blacklist needs analysis

### ISSUE THREAD

**Issue by austinstarin** [punkave/stylelint-config-punkave#17]

**Comment by abea** [comment]
What are you looking for in terms of analysis on these issues? A reasoned recommendation?

**Comment by austinstarin** [comment]
Yes. For example `selector-attribute-operator-blacklist` is something we could configure but we're currently not. We need to evaluate.

**Comment by abea** [comment]
I don't see a reason to use this rule. Would disallow particular attribute selectors, e.g., [target**="**_blank"]. Doesn't seem to be a problem needing to be solved.

**Comment by abea** [comment]
I wouldn't recommend we use this. Would disallow particular attribute selector operators, such as [target**="**_blank"]. Doesn't seem to be a problem needing a rule.
